clouddriver-elasticsearch/src/main/java/com/netflix/spinnaker/clouddriver/elasticsearch/ops/BulkUpsertEntityTagsAtomicOperation.java

```diff
@@ -246,8 +246,16 @@ private static void addTagMetadata(Date now, EntityTags entityTags) {
     entityTags.getTags().forEach(tag -> entityTags.putEntityTagMetadata(tagMetadata(tag.getName(), now)));
   }
 
-  private static AccountCredentials lookupAccountCredentials(AccountCredentialsProvider accountCredentialsProvider,
-                                                             String entityRefAccountId) {
+  private static AccountCredentials lookupAccountCredentialsByAccountName(AccountCredentialsProvider accountCredentialsProvider,
```

Comment:
I wasn't sure whether to do this this way or to simply use the `getCredentials` method and then handle a `null` returned - throwing an exception there.